.github/workflows/ci.yml

```diff
@@ -162,6 +167,19 @@ jobs:
     secrets:
       earthly_runner_address: ${{ secrets.earthly_runner_address }}
       earthly_runner_secret: ${{ secrets.earthly_runner_secret }}
+  nightly:
+    uses: ./.github/workflows/pages.yml
+    if: inputs.nightly == true
+    needs: [build, check, package, test]
+    with:
+      aws_role_arn: ${{ inputs.aws_role_arn }}
+      aws_region: ${{ inputs.aws_region }}
+      ci_cli_version: ${{ inputs.ci_cli_version }}
+      earthfile: ./docs
```

Comment:
as above you dont need to specify the target here, the workflow run will search for all the targets that start with `nightly` in the Earthfiles of the repo